core/commands/filestore.go

```diff
@@ -185,6 +186,107 @@ For ERROR entries the error will also be printed to stderr.
 	Type: filestore.ListRes{},
 }
 
+var fixFileStore = &cmds.Command{
```

Comment:
Given that this is 90% the same as `verify`, what if we just make it a flag on verify? `ipfs filestore verify --remove-bad-blocks`?